### PR TITLE
support negative values in nextSupplyInterestRate

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: contracts/connectors/loantoken/LoanTokenLogicStandard.sol [hunk 1]
```diff
@@ -437,19 +437,30 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension {
...
+        int256 supplyAmount)
         external
         view
         returns (uint256)
     {
         uint256 assetBorrow = totalAssetBorrow();
+        uint256 absSupplyAmount = abs(supplyAmount);
+        uint256 totalSupply = _totalAssetSupply(assetBorrow);
+
+        if(totalSupply<=absSupplyAmount)
```
Comment (RomanHiden) [inline contracts/connectors/loantoken/LoanTokenLogicStandard.sol]: format

**Review by Drypto13 — APPROVED** [review]
Looks good

**Review by RomanHiden — APPROVED** [review]

**Comment by RomanHiden** [comment]
@eugenesh1 please rebase to `interest-twap`

**Review by RomanHiden — APPROVED** [review]
looks good

**Pull request merged**